libraries/bxdf/usd_preview_surface.mtlx

```diff
@@ -363,7 +368,14 @@
       <input name="geomprop" type="string" interfacename="varname" />
       <input name="default" type="string" interfacename="fallback" />
     </geompropvalue>
-    <output name="out" type="string" nodename="primvar" />
+    <output name="out" type="string" nodename="primvar" uniform="true" />
+  </nodegraph>
+  <nodegraph name="IMP_UsdPrimvarReader_filename" nodedef="ND_UsdPrimvarReader_filename">
+    <geompropvalue name="primvar" type="string">
```

Comment:
I think because we're introducing an entirely new node `geompropvalueuniform` to handle strings - I think this node XML tag needs to match that new name - `geompropvalueuniform`. 

I also wonder if there are other cases elsewhere in the test suite where the existing `geompropvalue` node is being used for a `string` or `filename`, that would also need to be updated.